Formula/digdag.rb

```diff
@@ -0,0 +1,24 @@
+class Digdag < Formula
+  desc "Workload Automation System"
+  homepage "https://github.com/treasure-data/digdag"
+  url "https://dl.digdag.io/digdag-0.8.1.jar"
+  sha256 "92e79f5eb26b8992c9d510206c71f64fe4765c0c51a4e4cb62279aa1a4231ec7"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    # Add prefix a shebang before install for OS X 10.9.
+    # See https://github.com/Homebrew/homebrew-core/pull/2012#issuecomment-226407948 for the detail.
+    inreplace "digdag-#{version}.jar", /\A/, "#!/bin/sh\n"
```

Comment:
I suggested to add a shebang but it is rejected.
(I'm not sure about hybrid script mechanism...)
https://github.com/treasure-data/digdag/pull/125

My suggestion is to separate test by MacOS.version like below.

```ruby
  test do
    ENV.java_cache
    if MacOS.version > :mavericks
      assert_match version.to_s, shell_output("#{bin}/digdag --version")
    else
      assert (bin/"digdag").executable?
    end
  end
```